src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -82,19 +81,6 @@
     private final JobSchedulerService jobSchedulerService;
     private final CCDCreateCaseService ccdCreateCaseService;
 
-    private static BiFunction<ClaimSubmissionOperationIndicators, ClaimDocumentType, ClaimSubmissionOperationIndicators>
```

Comment:
NOTE: Cleaned up this BiFunction as it was never used as the assignment target for a lambda expression or method reference. Rather it was misused to muddle the code.